pkg/kube/config.yaml

```diff
@@ -8,3 +8,5 @@ log: "/var/lib/rancher/k3s/k3s.log"
 debug: true
 etcd-expose-metrics: true
 container-runtime-endpoint: "/run/containerd-user/containerd.sock"
+etcd-arg:
+  - "quota-backend-bytes=8589934592"
```

Comment:
Where does the 8GB number come from? Should it be documented? Should it change based on the size of something else?

Reply:
This is based off a suggested max here: https://etcd.io/docs/v3.4/dev-guide/limit/#storage-size-limit